sqldb-postgres/Makefile

```diff
@@ -2,7 +2,7 @@
 
 PROJECT = sqldb-postgres
 CAPABILITY_ID = "wasmcloud:sqldb"
-NAME = "SqlDb-Postgres"
+NAME = "SqlDb Postgres"
```

Comment:
Maybe Postgres SQL Provider ?

Reply:
I like `Postgres SQL` better, but I've been keeping `provider` out of the names. Open to reconsider

Reply:
Agreed we probably shouldn't put the word provider in a providers name